datamodel/referencedata.d/portcallservicetypecodes.csv

```diff
@@ -5,3 +5,5 @@ CRGO,Cargo operations,"The activity of discharging, shifting, loading and lashin
 TOWG,Towage,"The activity of one vessel (boat/tug) assisting another vessel with manoeuvres within the port area."
 BUNK,Bunkering,"The transfer of liquid or gaseous fuel from land-based or floating facilities into a ship’s permanent tanks"
 WSDP,Waste disposal,"The activity of removing waste from a vessel to shore during port stay"
+FAST,Fast,"FAST is all fast"
+GWAY,Gangway,"GWAY is gangway down and secure (authorized personnel can board the vessel)"

```

Comment:
I seem to recall there are 2 more values (LASH + one more that I don't remember)..

Reply:
LASH | Lashing | All activities encompassing securing the    containers and other arrangements onboard to prevent them from moving around    on the vessel at sea.

SAFE | Safety | A safety timestamps indicates an   acknowledgement of a participant that the vessel is ready for a responsibility   handover from carrier to terminal and otherwise.